templates/partials/about.html

```diff
@@ -10,6 +10,17 @@
         <li><a ng-click="reload()">{{'Reload' | translate}}</a></li>
       </ul>
     </div>
+    <div>
+      <h3>{{'initial.replication.title' | translate}}</h3>
+      <ul>
+        <li>{{'initial.replication.status' | translate}}: {{initialReplicationStatus}}</li>
+        <li ng-show="initialReplicationDuration">{{'initial.replication.duration' | translate}}: {{initialReplicationDuration / 1000 | number:1}} s</li>
+        <span ng-show="initialReplicationDataUsage">
+          <li>{{'android_app.data_usage.rx' | translate}}: {{initialReplicationDataUsage.rx | number}} B</li>
+          <li>{{'android_app.data_usage.tx' | translate}}: {{initialReplicationDataUsage.tx | number}} B</li>
+        </span>
+      </ul>
+    </div>
```

Comment:
Some of these translation keys aren't in medic-api, eg: "initial.replication.status". Add the default translations to messages-en.properties as part of this change.